test/build_model.py

```diff
@@ -537,9 +504,6 @@ def test_006_full_compilation_hummingbird_rf(self):
     def test_007_full_compilation_hummingbird_xgb(self):
         assert full_compilation_hummingbird_xgb()
 
-    def test_008_full_compile_individual_stages(self):
-        assert full_compile_individual_stages()
-
     def test_009_custom_stage(self):
```

Comment:
Nit: Please rename `test_009` and `test_010` to  `test_008` and `test_009`

Reply:
Wont fix; would need to rename half the tests in the file to make it consistent. The goal is to put the tests in a specific order, and that order is preserved in my existing changeset here.